lib/Intlc/Backend/TypeScript/Compiler.hs

```diff
@@ -25,8 +25,14 @@ compileNamedExport x l k v =
 compileTypeof :: InterpStrat -> ICU.Message -> Text
 compileTypeof x = let o = fromStrat x in flip runReader o . typeof . fromMsg o
 
-reactImport :: Text
-reactImport = "import React, { ReactElement } from 'react'"
+isTypeScriptReact :: Translation -> Bool
+isTypeScriptReact (Translation _m  be) = case be of
+  TypeScript      -> False
+  TypeScriptReact -> True
```

Comment:
Arguably a minor improvement would be to pattern match directly in the function parameter:

```hs
isTypeScriptReact (Translation _ TypeScriptReact) = True
isTypeScriptReact _ = False
```

Although that this function exists at all is another argument for lenses, or even just field selectors.

Reply:
Ah very cool!